src/components/Preview.vue

```diff
@@ -44,14 +44,16 @@ export default {
   methods: {
     onImageSampled(imageData) {
       this.imageData = imageData;
-      const draggableElement = document.querySelector('#draggable');
-      panzoom(draggableElement);
     },
     getColor(x, y) {
       const index = 4 * (y * this.imageData.width + x);
       const [r, g, b, a] = this.imageData.data.slice(index, index + 4);
       return `rgba(${r}, ${g}, ${b}, ${a / 256})`;
     },
+    mount() {
```

Comment:
This should be on the `mounted` hook, not a method. https://vuejs.org/v2/api/#mounted

Reply:
Ah, yep. Should be working in https://github.com/brick-a-pic/brick-a-pic/pull/27/commits/16bacffdee5639cd7e6e39c16599f6633ec810dc